app/processors/casino/login_credential_requestor_processor.rb

```diff
@@ -34,7 +34,8 @@ def handle_allowed_service
   end
 
   def handle_logged_in
-    if @listener.method(:user_logged_in).arity > 1
+    arity = @listener.method(:user_logged_in).arity
+    if arity > 1 or arity == -3
```

Comment:
-3 is the arity of the method that has 3 parameters and the last one has a default value. So originally the implementation was 'if arity > 1' which is ok for 3 parameters, but not for 3 parameters with a default value... The other way which could be used here would be 'if arity.abs > 1'...